src/Inputs/Checkboxes.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Jeremeamia\Slack\BlockKit\Inputs;
+
+use Jeremeamia\Slack\BlockKit\Exception;
+use Jeremeamia\Slack\BlockKit\Partials\Option;
+
+class Checkboxes extends InputElement
+{
+
+    use HasConfirm;
+
+    private const MIN_OPTIONS = 1;
+    private const MAX_OPTIONS = 10;
+
+    /** @var array|Option[] */
+    private $options = [];
+
+    /** @var array|Option[] */
+    private $initialOptions = [];
+
+    public function addOption(Option $option, bool $isInitial = false): Checkboxes {
+
+        $this->options[] = $option;
+
+        if ($isInitial) {
+            $this->initialOptions[] = $option;
+        }
+
+        return $this;
+
+    }
+
+    public function validate(): void
+    {
+
+        if (count($this->options) > self::MAX_OPTIONS) {
+            throw new Exception('Option Size cannot exceed %d', [self::MAX_OPTIONS]);
+        }
+
+        if (count($this->options) < self::MIN_OPTIONS) {
+            throw new Exception('Option Size must be at least %d', [self::MIN_OPTIONS]);
+        }
+
```

Comment:
You need additional validations:
```php
$this->validateOptions();

if (!empty($this->initialOptions)) {
    foreach ($this->initialOptions as $option) {
        $option->validate();
    }
}

if (!empty($this->confirm)) {
    $this->confirm->validate();
}
```

Reply:
initial options can only be part of the options array, they are already validated, still validate them again ?

Reply:
I see what you mean, but please do it for now. Thanks.